### PR TITLE
update faculty list of Hasso Plattner Institute.

### PR DESCRIPTION
**Pull request by yanghaojin**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -5330,7 +5332,6 @@ Ernst-Rüdiger Olderog,University of Oldenburg,https://uol.de/en/computingscienc
 Erol Sahin,Middle East Technical University,http://www.kovan.ceng.metu.edu.tr/~erol/Home.html,aO-849gAAAAJ
 Errol L. Lloyd,University of Delaware,https://www.eecis.udel.edu/~elloyd,805n5zYAAAAJ
 Ertan Onur,Middle East Technical University,https://eonur.ceng.metu.edu.tr/blog,6ybYW3QAAAAJ
-Erwin P. Bottinger,Hasso Plattner Institute,https://hpi.de/en/the-hpi/people/professors/prof-dr-ewin-boettinger.html,3dWDnnYAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: This is required. See the DBLP entry. https://dblp.org/pers/b/Bottinger:Erwin_P=.html

File: csrankings.csv [hunk 2]
```diff
@@ -1352,6 +1352,7 @@ Anisoara Calinescu,University of Oxford,http://www.cs.ox.ac.uk/people/ani.caline
 Anita Sarma,Oregon State University,http://eecs.oregonstate.edu/people/sarma-anita,O4TGxGMAAAAJ
 Anita Wasilewska,Stony Brook University,http://www3.cs.stonybrook.edu/~anita,NOSCHOLARPAGE
 Anja Feldmann [INF],Max Planck Society,https://www.mpi-inf.mpg.de/inet/people/anja-feldmann,mSK3340AAAAJ
+Anja Lehmann,Plattner Institute,https://hpi.de/das-hpi/personen/professoren/prof-dr-anja-lehmann.html,zH3fwYoAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:1355]: Typo in affiliation.

**Review by emeryberger — CHANGES_REQUESTED** [review]
Please address comments and I will merge.

File: csrankings.csv (hunk 1 above)
Comment (yanghaojin) [reply csrankings.csv]: I realized that there is another same entry for Prof. Bottinger in the file, and deleted the duplicated one.

File: csrankings.csv (hunk 2 above)
Comment (yanghaojin) [reply csrankings.csv:1355]: I already resolved this issue in this commit:
https://github.com/emeryberger/CSrankings/pull/2691/commits/7359505a08a4802b80b8e5f1ba440b8d22b449dd

File: csrankings.csv (hunk 1 above)
Comment (emeryberger) [reply csrankings.csv]: It's not duplicated. It's a different entry without an umlaut. It needs to be there. Look at the DBLP page and you will see *two* names.

**Pull request merged**